src/test/java/com/rultor/agents/github/CommentsTagTest.java

```diff
@@ -93,6 +94,32 @@ public void duplicatesRelease() throws Exception {
     }
 
     /**
+     * CommentsTag cannot release an older version.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void tooOldRelease() throws Exception {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        final Agent agent = new CommentsTag(repo.github());
+        final String tag = "1.5";
+        repo.releases().create("1.0");
+        repo.releases().create("2.0");
+        repo.releases().create("3.0-b");
+        final Talk talk = CommentsTagTest.talk(issue, tag);
+        agent.execute(talk);
+        final Comment.Smart response = new Comment.Smart(
+                repo.issues().get(1).comments().get(1)
+        );
+        MatcherAssert.assertThat(
+                response.body(),
+                    Matchers.containsString(
+                            "version tag is too low"
+                    )
```

Comment:
@caarlos0  What exactly is the issue with the format ( How can I fix it) ? FYI, This unittest method is is pretty much the same as the existing unittests in that class as it was copied.